examples/function/counter/example.py

```diff
@@ -0,0 +1,33 @@
+from typing import Iterator
```

Comment:
separate `map` and `reduce` as two examples?

Reply:
no, this map is not required.. committed it accidentally 

deleted